test/unit/continue_test.lua

```diff
@@ -0,0 +1,242 @@
+local expirationd = require("expirationd")
+local fiber = require("fiber")
+local t = require("luatest")
+local helpers = require("test.helper")
+local g = t.group('expirationd_continue')
+
+local task_name = "walk_all"
```

Comment:
You can pass this variable to evals too if you want consistency.

Reply:
I guess you mean something like that:
```patch
--- a/test/integration/reload_test.lua
+++ b/test/integration/reload_test.lua
@@ -62,7 +62,8 @@ local function reload_roles(srv)
     t.assert_equals({ok, err}, {true, nil})
 end
 
-local task_sleep_on_10_eval = [[
+local walk_task_name = "walk_all"
+local task_sleep_on_10_eval = string.format([[
     local fiber = require("fiber")
     local expirationd = require('expirationd')
 
@@ -78,7 +79,7 @@ local task_sleep_on_10_eval = [[
         end
         return true
     end
-    task = expirationd.start("walk_all", box.space.customers.id, is_expired_sleep,
+    task = expirationd.start("%s", box.space.customers.id, is_expired_sleep,
                              {process_expired_tuple = function() return true end,
                               force = true})
 
@@ -88,9 +89,9 @@ local task_sleep_on_10_eval = [[
         end
         fiber.yield()
     end
-]]
+]], walk_task_name)
 
-local task_first_tuple_eval = [[
+local task_first_tuple_eval = string.format([[
     local fiber = require("fiber")
     local expirationd = require('expirationd')
 
@@ -101,7 +102,7 @@ local task_first_tuple_eval = [[
         end
         return true
     end
-    task = expirationd.start("walk_all", box.space.customers.id, is_expired_tuple,
+    task = expirationd.start("%s", box.space.customers.id, is_expired_tuple,
                              {force = true})
 
     for i = 1,100 do
@@ -114,7 +115,7 @@ local task_first_tuple_eval = [[
     task:kill()
 
     return tuple or {}
-]]
+]], walk_task_name)
```

Reply:
This is a valid approach. The other one is
```lua
local eval_str = [[
    local arg = ...
    return func(arg)
]]

net_box.eval(eval_str, {arg})
```